src/test/resources/capabilities.yml

```diff
@@ -39,18 +38,8 @@ cloudDriver:
       browser_version: latest
       capabilities:
         browserstack.selenium_version: 3.141.59
-#    - name: OSX_BigSur_Chrome_Latest
-#      os: OS X
-#      os_version: Catalina
-#      browser: Safari
-#      browser_version: '13.0'
     - name: Android_Samsung Galaxy S21_11.0
       os: Android
       os_version: '11.0'
       device: Samsung Galaxy S21
       real_mobile: true
-#    - name: iOS_iPhone XS_13.0
-#      os: iPhone
-#      os_version: '13.0'
-#      device: iPhone XS
-#      real_mobile: true
```

Comment:
Shouldn't we add it back as uncommented.

Reply:
we could... wasn't sure on why we commented out. hence removed for now.